docs/api/cases/cases-api-create.asciidoc

```diff
@@ -0,0 +1,234 @@
+[[cases-api-create]]
+== Create case API
+++++
+<titleabbrev>Create case</titleabbrev>
+++++
+
+Creates a case.
+
+=== Request
+
+`POST <kibana host>:<port>/s/<space_id>/api/cases`
+
+=== Prerequisite
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
```

Comment:
Same as https://github.com/elastic/kibana/pull/127686/files#r829225831

Reply:
Done in 0cfb1fab28ae261de027ab7416387891fbfe1e26